src/components/Tabs/Tabs.jsx

```diff
@@ -2,0 +1,28 @@
+import { Tab } from '../Tab/Tab';
+
+export const Tabs = ({ tabs, findActive, active }) => {
+  const activeTab = tabs.find(tab => tab.id === active.id);
```

Comment:
Update the logic to use `activeTabId` instead of `active.id` to match the updated prop name.